storage/transactions.go

```diff
@@ -4,12 +4,16 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
+// TransactionsReader represents persistent storage read operations for transactions.
+type TransactionsReader interface {
+	// ByID returns the transaction for the given fingerprint.
```

Comment:
let's take the opportunity to improve the docs for these interfaces as we're making updates. please update these godocs with the errors as well.